docs/tutorials/how-to-navigate-k8s-resources-and-edit-the-source-code.md

```diff
@@ -0,0 +1,71 @@
+# How to Navigate K8s Resources and Edit the Source Code
```

Comment:
Let's call this "How to Navigate K8s Resource and edit their manifests"

Reply:
Done